packages/zk-eddsa-ticket-pcd/test/ZKEdDSATicketPCD.spec.ts

```diff
@@ -70,7 +74,8 @@ describe("EdDSA partial ticket should work", function () {
     timestampSigned: 1693028498280,
     attendeeSemaphoreId: identity.getCommitment().toString(),
     isConsumed: false,
-    isRevoked: false
+    isRevoked: false,
+    ticketCategory: TicketCategory.Devconnect
   };
 
```

Comment:
Same suggestion to add a test to verify the new field here, though this PCD is deprecated so it might be less important to keep it in parity if there's a lot of effort involved.